content/en/docs/concepts/overview/working-with-objects/finalizers.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Finalizers
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+In Kubernetes, finalizers are keys that allow you to specify pre-delete
+operations for resources. You can use finalizers to control garbage collection
+of resources by alerting controllers to perform specific cleanup tasks before
+deleting the target resource. 
+
+Finalizers don't usually specify the code to execute. Instead, they are
+typically lists of keys on a specific resource similar to annotations.
+Kubernetes specifies some finalizers automatically, but you can also specify
+your own.
+
+## How finalizers work
+
+When you create a resource using a manifest file, you can specify finalizers in
+the `metadata.finalizers` field. When you attempt to delete the resource, the
+controller that manages it notices the values in the `finalizers` field and does
+the following: 
+
+  * Modifies the object to add a `metadata.deletionTimestamp` field with the
+    time you started the deletion.
+  * Marks the object as read-only until the `finalizers` field is empty.
+
+The controller then attempts to satisfy the requirements of the finalizers
+specified for that resource. Each time a finalizer condition is satisfied, the
+controller removes that key from the resource's `finalizers` field. When the
+field is empty, garbage collection continues. You can also use finalizers to
```

Comment:
```suggestion
field is empty, {{< glossary_tooltip term_id="garbage-collection" text="garbage collection">}}
continues. You can also use finalizers to
```

Reply:
Ack - adding the tooltip in the first paragraph on this page.